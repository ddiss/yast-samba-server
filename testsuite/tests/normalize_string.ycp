{

// test for normalize_string
// $Id$

    include "testsuite.ycp";
    
    include "samba-server/routines.ycp";

    DUMP( normalize_string( "yes", nil ) );    
    DUMP( normalize_string( "yEs", nil ) );    
    DUMP( normalize_string( "1", nil ) );    
    DUMP( normalize_string( "true", nil ) );    
    DUMP( normalize_string( "TRue", nil ) );    
    DUMP( normalize_string( "no", nil ) );    
    DUMP( normalize_string( "0", nil ) );    
    DUMP( normalize_string( "FALSE", nil ) );
    DUMP( normalize_string( "auto", nil ) );
    DUMP( normalize_string( nil, "true" ) );
}
