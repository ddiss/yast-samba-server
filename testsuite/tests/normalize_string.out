Dump	yes
Dump	yes
Dump	yes
Dump	yes
Dump	yes
Dump	no
Dump	no
Dump	no
Dump	auto
Dump	true
